src/main/java/FileData.java

```diff
@@ -0,0 +1,19 @@
+public class FileData {
+
+    /**
+     * Name of file
+     */
+    public String Name;
+
+    /**
+     * Path to file
+     */
+    public String Path;
+
+    /**
+     * Size of file
+     */
+    public int Size;
```

Comment:
Usually, we doing
`private int size` and add getters and setters, constructors

Those classes we usually named POJO (in our community), but with additional strict. https://www.baeldung.com/java-pojo-class#javabeans without serialization